### PR TITLE
The Next Web Responsive

### PR DESCRIPTION
**Pull request by gasb150**
In this project, we clone the Next Web homepage using semantic HTML5 elements that are appropriately sized and positioned. We used flexbox, CSS grid, and positioning to position the various elements. We made the background with images and gradients. We use media queries to make the page responsive at two main breakpoints 1024px and 767px.

### CODE REVIEW

**Review by ElijahTobs — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle: 

## Hi @abongsjoel and @gasb150 

:+1: You've done a great job so far and that's commendable.
However, there are still some changes you need to effect before this project can be approved.

## [REQUIRED]

- [x] The screenshot in your README.md does not display

- [x] When you scroll down on the page, the second navbar alone should be sticky

- [x] On large and medium screen, increase the font size of 

Please effect the required changes and submit for another review

## Happy Coding :computer: 

_Feel free to reach out to me on slack_ **`@Elijah Ayandokun`**
  **Elijah O. Ayandokun**
[Github](https://github.com/ElijahTobs) | [LinkedIn](https://www.linkedin.com/in/ayandokunelijah/) | [Twitter](https://twitter.com/ElijahTobs)

**Comment by mimipeshy** [comment]
## STATUS : APPROVED :man_dancing:  :partying_face: 

Hi team :wave:  :wave: , congratulations on making all the required changes as pointed out by the TSE, you did great on this job, go ahead and merge. I wish you all the best :smile: 

Regards,
Peris Ndanu

**Pull request merged**
